index.bs

```diff
@@ -601,35 +601,35 @@ to follow these steps: [[!WSP]]
    :: a new {{DOMString}} containing |data|
 
    : |type| indicates that the data is Binary and {{WebSocket/binaryType}} is
-     "<dfn><code>blob</code></dfn>"
+     "<dfn>`blob`</dfn>"
```

Comment:
This should be `<dfn enum-value for=BinaryType>blob</dfn>` and then references can be `"{{BinaryType/blob}}"`.

Reply:
Weirdly the dfn part won't match the definition in the IDL unless you put the quotes inside the tags, ie. `<dfn enum-value for=BinaryType>"blob"</dfn>`. I'm not sure if this is a Bikeshed bug or WAI.